packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -29,6 +29,8 @@ import { PlaySubscriptions } from '../payments/google-play/subscriptions';
 import { PayPalHelper } from '../payments/paypal/helper';
 import { StripeHelper } from '../payments/stripe';
 import { AuthLogger, AuthRequest } from '../types';
+import doc from './utils/docs/account_docs';
```

Comment:
If we want to stick these in a `docs` directory, we might want to use the existing `fxa-auth-server/docs`, like `docs/swagger/` 🤔 

Also I think almost all of our new JS files are kebab-case, but I see we have a lovely mix of snake case for MD files.

Reply:
I have not heard it referred to kebab-case before (.... and now I am hungry). I'll be sure to update this. 😅